### PR TITLE
Update Danger for new SwiftLint config location

### PR DESCRIPTION
**Pull request by AvdLee**
Needed for the new BuildTools folder

### CODE REVIEW

File: Dangerfile.swift [hunk 1]
```diff
@@ -1,4 +1,4 @@
 import Danger
 import WeTransferPRLinter
 
-WeTransferPRLinter.lint(reportsPath: "WeTransferPRLinter/build/reports", swiftLintConfigsFolderPath: "SwiftLint")
+WeTransferPRLinter.lint(reportsPath: "WeTransferPRLinter/build/reports", swiftLintConfigsFolderPath: "BuildTools/SwiftLint")
```
Comment (kevinrenskers) [inline Dangerfile.swift:4]:
```suggestion
WeTransferPRLinter.lint(reportsPath: "WeTransferPRLinter/build/reports", swiftLintConfigsFolderPath: "BuildTools")
```

Comment (kevinrenskers) [reply Dangerfile.swift:4]: We don't have nested folders

**Comment by wetransferplatform** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Error deserializing SwiftLint JSON response (): dataCorrupted(Swift.DecodingError.Context(codingPath: [], debugDescription: "The given data was not valid JSON.", underlyingError: Optional(Error Domain=NSCocoaErrorDomain Code=3840 "No value." UserInfo={NSDebugDescription=No value.})))
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Testing cancelled because the build failed.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Swift package target 'Danger' is linked as a static library by 'WeTransferPRLinterTests' and 'Danger', but cannot be built dynamically because there is a package product with the same name.</td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  View more details on <a href="https://app.bitrise.io/build/e4a6d07a-9afe-44f1-b331-727e433a940a" target="_blank">Bitrise</a>
  </td>
    </tr>
  </tbody>
</table>

## Code Coverage Report
| Name | Coverage ||
| --- | --- | --- |

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/swift">Danger Swift</a> against ac01e9b2061a0dd252a154a7f13000b909bf2c5a
</p>

File: Dangerfile.swift (hunk 1 above)
Comment (AvdLee) [reply Dangerfile.swift:4]: Sharp!

**Review by kairadiagne — APPROVED** [review]

**Review by kevinrenskers — APPROVED** [review]

**Pull request merged**